ppl/cmd/zar/README.md

```diff
@@ -386,15 +406,15 @@ The classic map-reduce example is word count.  Let's do this example with
 the uri field in http logs.  First, we map each record that has a uri
 to a new record with that uri and a count of 1:
 ```
-zar map -o words.zng "uri != null | cut uri | put count=1"
+zar map -q -o words.zng "uri != null | cut uri | put count=1"
 ```
 again you can look at one of the files...
 ```
-find $ZAR_ROOT -name words.zng | head -n 1 | xargs zq -t -
+find $ZAR_ROOT -name words.zng ! -size 0 | head -n 1 | xargs zq -z -
```

Comment:
The `! -size 0` became necessary because, with the new distribution of records across chunks, some of the chunks completely lack events containing the `uri` field, hence some zero-length `words.zng` files end up in the tree, and we want to avoid those.